react-commenting.js

```diff
@@ -29,15 +38,25 @@ class App extends Component{
         <h3>Enter text here:</h3>
         <input
           {/* 4. */}
+          //
           onChange={ this.robot }
```

Comment:
here we call the `robot` method above every time there is a change inside the input box